### PR TITLE
Modify settings for npm publish

### LINKED ISSUE #46: Modify the settings for npm publish.
**What you were expecting:**

A `lib` folder is created in each folder and the `.js` file needs to be published.

**What happened instead:**

When publishing npm, currently the `lib` folder is in the root folder, so only the `.ts` files are published.

**Steps to reproduce:**

**Related code:**

* A link to a GitHub repo with the minimal codebase to reproduce the issue

```
insert short code snippets here
```

**Other information:**

**Environment** 

* 0Auth version:
* Last version that did not exhibit the issue (if applicable):
* Node version:
* Browser:
* Stack trace (in case of a JS error):

### PR DESCRIPTION
**Pull request by HyeockJinKim**
# Summary 

Add lerna and modify package.json and tsconfig.json

- It Fixes #46

## How to Review? 

- Please refer to my self-review submitted as the review.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,22 +1,20 @@
 {
   "compilerOptions": {
-    "target": "ES2015",
-    "module": "ES2015",
-    "lib": ["es5", "es2015", "dom"],
+    "target": "ES5",
+    "module": "commonjs",
```
Comment (HyeockJinKim) [inline tsconfig.json:4]: Currently we build according to `ES5` rules.
It is created in `commonjs` format for use in general `js`.

File: tsconfig.json [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
     "strict": true,
     "baseUrl": ".",
-    "rootDir": ".",
-    "outDir": "lib",
     "paths": {
       "@0auth/*": ["packages/*/src"]
     },
     "declaration": true,
+    "composite": true,
```
Comment (HyeockJinKim) [inline tsconfig.json:11]: Options for reference.

**Review by HyeockJinKim — COMMENTED** [review]
self review

**Comment by HyeockJinKim** [comment]
npm link is [https://www.npmjs.com/settings/0auth/packages](https://www.npmjs.com/settings/0auth/packages) .

**Review by anthonyminyungi — APPROVED** [review]
Appreciated your effort.

**Pull request merged**